enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,367 @@
+libvirt.
+- Currently, the Multi-Arch team is using a development configuration (IPI on libvirt) to cover our CI workloads
+for OpenShift on ppc64le verification. This would help diversify and scale our CI effort.
+- Power hardware is especially targeted to AI and secure workloads, so building out cloud capabilities will help expand
+our workload footprint.
+- IPI is generally easier to install and set up quickly for demos and proof of concepts which can help lower the barrier
+of entry for Power-optimized workloads.
+
+### Goals
+
+- Provide a way to install OpenShift on Power VS infrastructure using the OpenShift installer with IPI provisioned
+against IBM Cloud APIs.
+- Implement the machine-api controllers to provide scaling and managing the cluster nodes post-install.
+
+### Non-Goals
+
+- UPI support. We've already documented how to install on pp64le hardware using UPI.
+
+## Proposal
+
+To leverage the Power VS platform, we will make use of the Power VS support in the [IBM Cloud Terraform
+Provider][ibm-cloud-terraform-provider]. This provider will interface with IBM Cloud APIs and ultimately enable the
+OpenShift Installer to provision VM resources using Power VS, which will be used as cluster nodes.
+Unlike some of the existing providers, this depends on CoreOS availability in the Power VS cloud in the OVA
+disk image format.
+
+The expected workflow would be as follows:
+1. The installer is invoked with an environmental variable API key to authenticate to the IBM Cloud.
+2. Once invoked, the installer interacts with a terraform provider to create a new Power VS service.
+3. The installer or terraform provider will then import a CoreOS OVA disk image file into the Power VS service from IBM
+Cloud Object Storage, which creates a corresponding boot image and makes it available to the Power VS service.
+4. Once imported the installer can start building the private network for the nodes.
+5. The bootstrap and master nodes will be created in the private network. These are called Power VS instances, and
+they get created using the CoreOS boot image.
+6. To expose this private network and provide load balancing, the IBM Cloud APIs will leverage the IBM Cloud
+[Direct Link][ibm-direct-link-website] offering. This will allow for both the ingress load balancer and the cluster nodes to
+talk over the private network on a high-speed link.
+7. Bootstrapping completes as normal.
+8. The workers will be added using the openshift/[cluster-\*machine-\*] controllers and operators based on the OpenShift
+MAPI model. A Power VS controller needs to be implemented and hooked into the [OpenShift Machine
+Operator][openshift-machine-operator].
+9. The cluster teardown can make use of the existing [go client for Power][power-go-client] via `openshift-install
+destroy cluster`.
+
+### User Stories
+
+- A user of this installer would like to see a cluster running within an hour of kicking off the installer.
```

Comment:
Since this story has a time limit maybe specify a default (3 master-2 worker) cluster?